packages/apps/emails/src/firebase.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/prefer-default-export */
+import '@cloudcommerce/firebase/lib/init';
+import config from '@cloudcommerce/firebase/lib/config';
+import functions from 'firebase-functions';
+import {
+  createAppEventsFunction,
+  ApiEventHandler,
+} from '@cloudcommerce/firebase/lib/helpers/pubsub';
+import handleApiEvent from './events-to-app-emails';
+import handleAbandonedCarts from './functios-lib/abandoned-carts';
+
+const { httpsFunctionOptions: { region } } = config.get();
+
+const functionBuilder = functions
+  .region(region)
+  .runWith({
+    timeoutSeconds: 300,
+    memory: '128MB',
```

Comment:
```suggestion
    memory: '256MB',
```

128 é só pra função que checa eventos mesmo, executa a cada minuto e é mais simples também, aqui é possível que 128 seja pouco demais e dê BO 🙃